examples/templates/cluster-operator/connect-template.yaml

```diff
@@ -96,3 +96,7 @@ objects:
         "offset.storage.replication.factor": "${KAFKA_CONNECT_OFFSET_STORAGE_REPLICATION_FACTOR}",
         "status.storage.replication.factor": "${KAFKA_CONNECT_STATUS_STORAGE_REPLICATION_FACTOR}"
       }
+    metrics-config: |-
+      {
+        "lowercaseOutputName": true
+      }
```

Comment:
should we add this example to the kafka-connect YAML files under /examples/configmaps/cluster-operator as well ?

Reply:
Done